index.js

```diff
@@ -27,9 +28,11 @@ function processNodeContentWithPosthtml(node, options) {
  * @param   {String}    locals  [string to parse as locals object]
  * @return  {Function}          [Function containing evaluated locals, or empty object]
  */
-function parseLocals(locals) {
+function parseLocals(attributeLocals = {}, optionLocals = {}) {
   try {
-    return expressions({locals: JSON.parse(locals)});
+    const locals = merge(JSON.parse(attributeLocals), optionLocals);
```

Comment:
It seems to me that the attribute should have higher priority when resolving values, such as inline styles have higher priority

Reply:
Done, switched the merging order so that attribute locals will be merged on top of option locals.

Reply:
Is it possible to add a footnote to the description about this?

Reply:
Done, rephrased the description to make it clear.